RS.Fritz.Manager.UI/App.xaml

```diff
@@ -789,6 +789,41 @@
                     </Expander>
                 </Border>
             </DataTemplate>
+            <DataTemplate DataType="{x:Type domain:HostsGetHostListPathResponse}">
+                <Border Background="{StaticResource ForegroundSolidColorBrush}" BorderBrush="{StaticResource TitleBackgroundSolidColorBrush}" BorderThickness="5" CornerRadius="10" Padding="5">
+                    <Expander Foreground="{StaticResource TextSolidColorBrush}" IsExpanded="True" Header="HostListPath">
+                        <Grid>
+                            <Grid.Resources>
```

Comment:
You can remove this resource and databind to the ObservableCollection directly:

DataGrid Grid.Row="3" Grid.Column="1" Height="150" IsReadOnly="true" ItemsSource="{Binding DeviceHostsCollection}"